.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .DS_Store*
 .hubot_history
+*.sw[o|p]
```

Comment:
Yeah I just realized that swo files are made if there is already an swp and it goes on to swn and so on. You can probably remove that line.

[]http://stackoverflow.com/questions/5039545/what-are-swo-and-swp-files-in-jboss-log-dir